### ISSUE TITLE
Drawer screen not accessible when drawerType is set to permanent

### ISSUE THREAD

**Issue by msuma268** [react-navigation/react-navigation#7976]
**Current Behavior**
I have a drawer navigator with 3 Screens and drawerType set as **permanent**. 
When I turn ON Accessibility (TalkBack/VoiceOver), I am able to **access only the drawer items list**. Even though Screen component gets updated based on the option selected, I am unable to access Screen component and read out its contents.
If drawerType is set as "front" instead of "permanent", able to access both drawer items list and Screen components.

- What code are you running and what is happening?
`//@ts-check
import { createDrawerNavigator } from '@react-navigation/drawer';
import { NavigationContainer } from '@react-navigation/native';
import { createStackNavigator } from '@react-navigation/stack';
import * as React from 'react';
import { Text, View } from 'react-native';

const Stack = createStackNavigator();
function Option3() {
  return (
    <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>
      <Text>Option 3!</Text>
    </View>
  );
}
function Option2() {
  return (
    <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>
      <Text>Option 2!</Text>
    </View>
  );
}

function Screen1() {
  return (
    <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center', backgroundColor: "blue" }}>
      <Text>Screen 1 !</Text>
    </View>
  );
}

function Option1Navigator() {
  return (
    <Stack.Navigator initialRouteName="Screen1"
      screenOptions={{
        headerTitleAlign: "center", headerStyle: { backgroundColor: "black" },
        headerTintColor: '#fff'
      }}>
      <Stack.Screen name="Screen1" component={Screen1} options={{ headerTitle: "Screen1" }} />
    </Stack.Navigator >
  );
}

const Drawer = createDrawerNavigator();

export default function App() {
  return (
    <NavigationContainer>
      <Drawer.Navigator drawerType="permanent" drawerStyle={{
        backgroundColor: '#c6cbef',
        width: 150
      }}>
        <Drawer.Screen name="Option1" component={Option1Navigator} />
        <Drawer.Screen name="Option2" component={Option2} />
        <Drawer.Screen name="Option3" component={Option3} />
      </Drawer.Navigator>
    </NavigationContainer>
  );
}`

- Include a screenshot or video if it makes sense.

**Expected Behavior**

- What do you expect should be happening?
Both drawer items list and screen component should be accessible when drawerType is set to "permanent"

- Include a screenshot or video if it makes sense.

**How to reproduce**

- You must provide a way to reproduce the problem. If you are having an issue with your machine or build tools, the issue belongs on another repository as that is outside of the scope of React Navigation.
- Either re-create the bug on [Snack](https://snack.expo.io) or link to a GitHub repository with code that reproduces the bug.
- Explain how to run the example app and any steps that we need to take to reproduce the issue from the example app.
- Keep the repro code as simple as possible, with the minimum amount of code required to repro the issue.
- Before reporting an issue, make sure you are on latest version of the package.

**Your Environment**

| software                       | version |
| ------------------------------ | ------- |
| iOS or Android                 | Both
| @react-navigation/native       |5.1.4
| @react-navigation/drawer       |5.4.0
| react-native-reanimated        |1.8.0
| react-native-gesture-handler   |1.6.1
| react-native-safe-area-context |0.7.3
| react-native-screens           |2.4.0
| react-native                   |0.62
| expo                           |
| node                           |10.18
| npm or yarn                    |npm
